packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1052,8 +1052,60 @@ export function isBStackSession(config: Options.Testrunner) {
     return false
 }
 
-export function shouldAddServiceVersion(config: Options.Testrunner, testObservability?: boolean): boolean {
-    if (config.services && config.services.toString().includes('chromedriver') && testObservability !== false) {
+export function isBrowserstackInfra(config: BrowserstackConfig & Options.Testrunner, caps?: Capabilities.BrowserStackCapabilities): boolean {
```

Comment:
wdio checks based on length of user and key to check for browserstack, why can't we use same logic?

Reply:
Didn't see this logic for O11y.

Reply:
Need to check if we can use isBstackSession